HISTORY.rst

```diff
@@ -3,6 +3,12 @@
 Release History
 ===============
 
+0.10.8
++++++++++++++++
+**IoT Central fixes**
+* Align SSL certificate verification with core behavior
```

Comment:
I would also include the specific configuration value the user can expect to be aligned i.e. the env var `AZURE_CLI_DISABLE_CONNECTION_VERIFICATION` would be honored.

Reply:
added